src/main.js

```diff
@@ -3,22 +3,17 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
-import Vue from 'vue'
+import { createApp, defineAsyncComponent } from 'vue'
 
 // Styles
 import './styles/styles.scss'
 import '@nextcloud/dialogs/style.css'
 
-Vue.prototype.t = t
-Vue.prototype.n = n
-Vue.prototype.OC = OC
-Vue.prototype.OCA = OCA
-
-export default new Vue({
-	el: '#notifications',
-	name: 'NotificationsApp',
-	components: {
-		NotificationsApp: () => import(/* webpackPreload: true */'./NotificationsApp.vue'),
-	},
-	render: (h) => h('NotificationsApp'),
-})
+const NotificationsApp = defineAsyncComponent(() => import(/* webpackPreload: true */'./NotificationsApp.vue'))
+export default createApp(NotificationsApp)
+	.use((app) => {
+		app.config.globalProperties.OC = window.OC
+		app.config.globalProperties.OCA = window.OCA
+		app.config.globalProperties.OCP = window.OCP
+	})
```

Comment:
Could you check that this is actually needed? I don't remember we still use it anywhere...

Reply:
@nickvergessen I didn't understand what you mean and if it is related to this code fragment...

This code adds global variables to every Vue component instance's properties in this specific Vue app.

Reply:
That's to be use in component/template? Otherwise it's available in global or window, right?